docs/testing-with-simulator.md

```
@@ -0,0 +1,253 @@
+curl -sk \
+  -H "Content-Type: application/json" \
+  -H "x-api-key: $API_KEY" \
+  "$GATEWAY/v1/chat/completions" \
+  -d '{
+    "model": "nonexistent-model",
+    "messages": [{"role": "user", "content": "hello"}],
+    "max_tokens": 50
+  }'
+```
+
+## Simulator Endpoint
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)